.github/workflows/ci.yml

```diff
@@ -2,9 +2,8 @@ name: CI
 
 on:
   push:
-    branches:
-      - '*'
-      - '!main'
```

Comment:
this change fixes the issue where CI doesn't run on branches with "/" in the name. Not sure how it fixes things, but it does